# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../motor/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

An ugly patch for an ugly lvalue cast - for gcc-3.4.

  - Rene Rebe <rene@exactcode.de>

--- ./src/ui/ncurses/uitagbrowser.cc.vanilla	2004-07-05 14:13:59.000000000 +0200
+++ ./src/ui/ncurses/uitagbrowser.cc	2004-07-05 14:08:51.000000000 +0200
@@ -78,7 +78,8 @@
     cm.setwindow(textwindow(w.x1, w.y1, w.x1+20, 0, uiconf.getcolor(cp_menufr)));
 
     for(motortagbrowser::viewscope scope = motortagbrowser::File;
-    scope != motortagbrowser::viewscope_size; (int) scope += 1) {
+    scope != motortagbrowser::viewscope_size;
+    scope = (motortagbrowser::viewscope) ((int)scope + 1) ) {
 	cm.additem(0, scope, (string) " " + getscopename(scope));
 	if(scope == tagbrowser.getscope())
 	    cm.setpos(cm.getcount()-1);
